content/learn/accessibility/code-review/index.md

```diff
@@ -28,7 +28,10 @@ Ensure that link text describes the target and purpose and that it can be distin
 
 ## Semantic markup
 
-Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example, <code>&lt;ul&gt;</code> unordered lists are used for lists, <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings, <code>&lt;nav&gt;</code> is used for navigation.
+Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example:
+- <code>&lt;ul&gt;</code> are used for unordered lists
+- <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings
+- <code>fieldset</code> and <code>legend</code> are used to [group related sets of controls](https://www.w3.org/WAI/tutorials/forms/grouping/).
 
```

Comment:
Should we have a fourth bullet for the nav element for navigation? Or did this change?